File: android/sited/src/main/java/org/noear/sited/SdNodeFactory.java
package org.noear.sited;

import android.app.Application;

/**
 * Created by yuety on 16/2/1.
 */
public class SdNodeFactory {
    public SdNode createNode(SdSource source) {
        return new SdNode(source);
    }

    public SdNodeSet createNodeSet(SdSource source) {
        return new SdNodeSet(source);
    }
}

File: ios/sited/sited/src/SdNodeFactory.swift
//
//  SdNodeFactory.swift
//  sited
//
//  Created by 谢月甲 on 16/2/4.
//  Copyright © 2016年 noear. All rights reserved.
//

import Foundation

public class SdNodeFactory {
    
    public init(){
        
    }
    
    public func createNode(source:SdSource)->SdNode!
    {
        return SdNode(source: source);
    }
    
    public func createNodeSet(source:SdSource)->SdNodeSet!
    {
        return SdNodeSet(source: source);
    }
}